Add documentation for new envs in serverless functions (#10943)
kyma-project/kyma: docs/serverless/05-01-serverless-envs.md

To configure the Function with the Node.js runtime, override the default values of these environment variables:

| Environment variable             | Description                                                                                                                  | Type    | Default value |
| -------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- | ------- | ------------- |
| **FUNC_TIMEOUT**                 | Specifies the number of seconds in which a runtime must execute the code.                                                    | Number  | `180`         |
| **REQ_MB_LIMIT**                 | Specifies the payload body size limit in megabytes.                                                                          | Number  | `1`           |

> | **KYMA_INTERNAL_LOGGER_ENABLED** | Enables default HTTP request logger, which uses standard Apache combined log output. To enable set this to any truthy value. | Boolean | `false`       |

Reviewer: ```suggestion
| **KYMA_INTERNAL_LOGGER_ENABLED** | Enables the default HTTP request logger which uses the standard Apache combined log output. To enable it, set its value to `true`.  | Boolean | `false`       |
```

To configure a Function with the Python runtime, override the default values of these environment variables:

| Environment variable             | Description                                                                                                                  | Unit    | Default value   |
| -------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- | ------- | --------------- | ------------------------------------------------------------------------------ |

> | **FUNC_MEMFILE_MAX**             | Maximum size of memory buffer for the HTTP request body in bytes.                                                            | Number  | `100*1024*1024` | <!-- https://bottlepy.org/docs/dev/api.html#bottle.BaseRequest.MEMFILE_MAX --> |

Reviewer: ```suggestion
| **FUNC_MEMFILE_MAX**             | Specifies the maximum size of the memory buffer for the HTTP request body in bytes.                                                            | Number  | `100*1024*1024` | <!-- https://bottlepy.org/docs/dev/api.html#bottle.BaseRequest.MEMFILE_MAX --> |
```

Reviewer: I unified the descriptions to start with the verb.

> | **CHERRYPY_NUMTHREADS**          | Number of requests that can be handled in parallel                                                                           | Number  | 10              |

Reviewer: ```suggestion
| **CHERRYPY_NUMTHREADS**          | Specifies the number of requests that can be handled in parallel                                                                           | Number  | 10              |
```

> | **KYMA_INTERNAL_LOGGER_ENABLED** | Enables default HTTP request logger, which uses standard Apache combined log output. To enable set this to any truthy value. | Boolean | `false`         |

Reviewer: ```suggestion
| **KYMA_INTERNAL_LOGGER_ENABLED** | Enables the default HTTP request logger which uses the standard Apache combined log output. To enable it, set its value to `true`. | Boolean | `false`         |
```
